charts/rancher-backup/files/default-resourceset-contents/rancher.yaml

```diff
@@ -52,3 +52,11 @@
   kindsRegexp: "^clusterrepos$"
 - apiVersion: "resources.cattle.io/v1"
   kindsRegexp: "^ResourceSet$"
+- apiVersion: "v1"
+  kindsRegexp: "^secrets$"
+  namespaceRegexp: "."
```

Comment:
should this be `^.*$`

Reply:
I believe you're right, I was following the other regex in the resourceSet that used a `.` to capture all kinds.

Reply:
Made your change and tested, found four test secrets in four different namespaces in the backup. So looks like it works!